src/types/bitcoin.ts

```diff
@@ -0,0 +1,81 @@
+import { HexString } from "./encoding";
+
+export type BitcoinNetwork = "mainnet" | "testnet" | "regtest";
+
+export type BlockHeader = {
+    merkleRoot: Uint8Array;
+    target: Uint8Array; // u256
+    timestamp: number;
+    version: number;
+    hash_: Uint8Array;
+    hashPrevBlock: Uint8Array;
```

Comment:
We are using `Uint8Array`, `string` and `HexString`, which afaik are all representing the same thing: an array of bytes. For consistency, can we refactor to use only a single type?

Reply:
Very good suggestion, I changed the types to use uniformly `HexString`. I would prefer `Buffer` as that's native type, but it was not possible because polkadotjs cannot convert it properly for some cases of witness data.